docs/Migrations.md

```diff
@@ -0,0 +1,35 @@
+# Managing migrations for Builder services
+
+All builder migrations are run with Diesel. This document describes how to create and manage those migrations.
```

Comment:
Super minor, but might be nice to link off to Diesel for those new to it (as I was).